### PR TITLE
Increase verify-generated-code step memory/CPU resources

### PR DESCRIPTION
**Pull request by omertuc**
Some performance improvements means we're using more CPU / memory,
to avoid the job getting [OOMKilled](https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_assisted-service/4589/pull-ci-openshift-assisted-service-master-edge-verify-generated-code/1589680483808055296), we need more 

https://github.com/openshift/assisted-service/pull/4589

### CODE REVIEW

**Comment by droslean** [comment]
/hold
@omertuc I am holding this PR to gather some data about how many resources the job needs. Did you check that already?

**Comment by omertuc** [comment]
> /hold @omertuc I am holding this PR to gather some data about how many resources the job needs. Did you check that already?

I haven't. However, the more logical core the container thinks it has (`nproc`), the more RAM it will use. So it's a bit hard to tell

**Comment by droslean** [comment]
We have https://pod-scaler-ui-ci.apps.ci.l2s4.p1.openshiftapps.com/usage/pods?branch=master&container=test&org=openshift&repo=assisted-service&target=edge-verify-generated-code&variant= to measure it. I rerun the jobs to gather more data.

**Comment by omertuc** [comment]
Thank you, updated the values to the recommended values

**Comment by omertuc** [comment]
Please unhold if you think it's ok;

File: ci-operator/step-registry/assisted/verify-generated-code/assisted-verify-generated-code-ref.yaml [hunk 1]
```diff
@@ -5,9 +5,9 @@ ref:
   commands: assisted-verify-generated-code-commands.sh
   resources:
     requests:
-      cpu: "3"
-      memory: 600Mi
+      cpu: "4"
+      memory: 1650Mi
```
Comment (droslean) [inline ci-operator/step-registry/assisted/verify-generated-code/assisted-verify-generated-code-ref.yaml:9]: This is the same as the limit.

Comment (omertuc) [reply ci-operator/step-registry/assisted/verify-generated-code/assisted-verify-generated-code-ref.yaml:9]: Is that bad?

Comment (omertuc) [reply ci-operator/step-registry/assisted/verify-generated-code/assisted-verify-generated-code-ref.yaml:9]: I thought that was the ideal (Guaranteed QoS class etc)

**Pull request merged**